language-pages/typesetting.md

```diff
@@ -8,6 +8,60 @@ sidenav: language-side-nav
+    - **line height**: a unitless multiple of space between lines, like 1.5
+- The reason it works this way is so different typefaces take up roughly the same amount of space within the page.
+- In order to have more control over how your type is displayed in the browser, you need to do more than pick a typeface and a pixel size. You should define each of your type elements (headings, paragraphs, lists, captions, etc.) using these 3 attributes.
+
+## Specifying a font family
+
+- USWDS has a set of [default font stacks](https://designsystem.digital.gov/design-tokens/typesetting/font-family/) that are named using a single token. For example, the token `merriweather` will result in this font stack: `"Merriweather Web", "Georgia", "Cambria", "Times New Roman", "Times", serif`.
+    - This means that if the Merriweather font files are not available for some reason, the browser will display the text using Georgia instead. If not Georgia, then Cambria, and so on.
+    - This is important to think about for the experience of a page loading as well. Font files can get large, and the browser will try to display text as soon as possible. So it uses the fallback fonts to render text and then changes it when the preferred font is available. If these fonts are very different in size or style, it can be jarring for the visitor.
+- If you want to use a typeface outside of the defaults, consider adding it on top of one of these default stacks. For example, if you want to use Proxima Nova, you can specify a custom stack built on top of `source-sans-pro`. So the developer will know you want this: `"Proxima Nova", "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif`.
+
+## Specifying font sizes
+
+- USWDS includes [theme tokens](https://designsystem.digital.gov/design-tokens/typesetting/font-size/#theme-tokens) for font sizes that use semantic, "t-shirt size" names: `sm`, `md`, `lg`, etc.
+- There are 9 sizes available in the default token set, ranging from 3xs to 3xl.
+- Unless you need more than 9 unique sizes in your project, consider specifying your sizes using these theme tokens. You can redefine what those default sizes using the [system token](https://designsystem.digital.gov/design-tokens/typesetting/font-size/#system-tokens) references (or defining your own custom sizes if you prefer).
+
+## Specifying line height
+
+- USWDS includes 6 [line height tokens](https://designsystem.digital.gov/design-tokens/typesetting/line-height/).
+- Consider limiting the number of line heights you use and redefine these tokens as needed. That way `line-height-3` means the same thing throughout your design and the vertical rhythm of your content will feel more cohesive.
+- Line height may be affected by component-specific styles, so if you're not seeing the expected results, it may be because of some embedded CSS rules that are taking precedence over what you intended for a particular element.
```

Comment:
Is it possible to see an example of this?

Reply:
You kind of see this with prose, but I can try to think of a more specific example.

Reply:
Couldn't quickly find another example other than Prose so leaving this out since there's a whole section for that.